src/components/trainingInput/TrainingInput.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { ChangeEvent, FormEvent } from "react";
+import { useState } from "react";
+
+/* type DropdownProps = {
+  label: string;
+  idSelect: string;
+  options : OptionProps[];
+};*/
+
+/* type TrainingProbs = {
+  label: string;
+  value: number;
+}; */
+
+function TrainingInput() {
+  const [training, setTraining] = useState({
+    sets: 1,
+    reps: 1,
+    weight: 1,
+  });
+
+  const handleOnSubmit = (event: FormEvent) => {
+    event.preventDefault;
+    console.log(event);
+  };
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    const value = event.target.value;
+    setTraining({ ...training, [event.target.name]: value });
+    console.log(training);
+  };
+
+  return (
+    <form onSubmit={handleOnSubmit}>
+      <label htmlFor="sets">Sets</label>
```

Comment:
This goes for all 3 labels:

```suggestion
      <label className={styles.label} htmlFor="sets">Sets</label>
```